lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1465,7 +1477,8 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
 
   void _handleParseCalendarEventSuccess(ParseCalendarEventSuccess success) {
     emailLoadedViewState.value = Right<Failure, Success>(success);
-    calendarEvent.value = success.calendarEventList.first;
+    calendarEvent.value = success.calendarEventMap.values.first.first;
+    _displayingEventBlobId = success.calendarEventMap.keys.first;
```

Comment:
Please make sure `calendarEventMap` is not empty.

Reply:
It is checked from `lib/features/email/domain/usecases/parse_calendar_event_interactor.dart` line `43 ~ 44`